smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -790,7 +805,19 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
             saveArgs.allArgs);
         break;
       }
-    }
+    },
+
+    // Test that GetAttrib succeeds for the `SCARD_ATTR_ATR_STRING` argument.
+    'testGetAttrib': async function() {
+      expectReportDataResult(
+          /*requestId=*/ 111, SimulationConstants.COSMO_ID_70_ATR, 'SUCCESS');
```

Comment:
nit: Comment `/*resultCode=*/ 'SUCCESS'`, as it's not self-explanatory here.

Reply:
done